src/xcode/ENA/ENA/Source/Scenes/HealthCertificates/Validation/Result/HealthCertificateTechnicalValidationFailedViewModel.swift

```diff
@@ -17,15 +17,15 @@ struct HealthCertificateTechnicalValidationFailedViewModel: HealthCertificateVal
 	}
 
 	// MARK: - Internal
+	
+	var title = AppStrings.HealthCertificate.Validation.Result.TechnicalFailed.title
```

Comment:
UX requested a scrolling title in this screen - so this might break this request